library/psa_crypto.c

```diff
@@ -3469,7 +3469,8 @@ psa_status_t psa_cipher_encrypt( mbedtls_svc_key_id_t key,
     status = psa_driver_wrapper_cipher_encrypt(
         &attributes, slot->key.data, slot->key.bytes,
         alg, local_iv, default_iv_length, input, input_length,
-        output + default_iv_length, output_size - default_iv_length,
+        ( output == NULL ? NULL : output + default_iv_length ),
```

Comment:
I'm sure I've talked with someone about debugging this, but it's not fixed in development. Maybe it's in another pending PR?

We really need to get around running a recent UBSan on the CI.